openedx/features/partners/helpers.py

```diff
@@ -103,4 +103,4 @@ def get_partner_from_user(user):
 
 
 def user_has_performance_access(user, partner):
-    return bool(partner and user.has_perm(PERFORMANCE_PERM_FRMT.format(slug=partner.slug)))
+    return bool(partner and user.has_perm('partners.%s' % PERFORMANCE_PERM_FRMT.format(slug=partner.slug)))
```

Comment:
use format at first place as well to be consistent

Reply:
Nice suggestion @zeshanarifios but to use format inside format was confusing and was not that much readable, so that's why I avoided nested format.

Reply:
But we can avoid nesting as well by updating our PERFORMANCE_PERM_FRMT format.